chapters/jina_box/index.md

```diff
@@ -0,0 +1,178 @@
+## Jinabox
+
+##### Jinabox is a lightweight, customizable omnibox. You can use it for searching text, images, videos, audio or any kind of data with a Jina backend.
+
+![jina box](jinabox.gif)
+
+
+### Features
+
+- Easy-to-use web component for Jina search backend
+- Users can drag and drop multi-modal data as search queries
+- Webcam and microphone support (in progress, contributions welcome! ❤️)
```

Comment:
Are we listing in progress features @FionnD?

Reply:
Emm... in general I would not. It's likely we'll forget to update the page in the future and then it's out of sync.